res/static/fragment_callback.js

```diff
@@ -0,0 +1,14 @@
+document.addEventListener('DOMContentLoaded', function() {
```

Comment:
If we use DOMContentLoaded, we run into the danger that that event might have already fired when we the JS is read. Sure, should only happen when including it async, but actually that is something we should do do gain a few performance pennys. Workaround is to check whether the event has already fired, as in https://github.com/onli/feedtragon/blob/161373ac3acd3738950e4bd4b0dacadd6eec4ccf/public/main.js#L431. I think that should be added.